runtimes/peregrine/src/tests.rs

```diff
@@ -41,15 +41,18 @@ use runtime_common::{
 
 use super::{Runtime, RuntimeCall};
 
-#[test]
-fn call_size() {
-	assert!(
-		core::mem::size_of::<RuntimeCall>() <= 240,
-		"size of Call is more than 240 bytes: some calls have too big arguments, use Box to reduce \
-		the size of Call.
-		If the limit is too strong, maybe consider increase the limit to 300.",
-	);
-}
+// TODO: Uncomment if pallet_assets implements measures to reduce their `Call`
```

Comment:
Is there already a PR? If yes, maybe link it here.

Reply:
No, there's not one, as far as I could find.